WBooks/Screens/Library/LibraryViewModel.swift

```diff
@@ -23,4 +32,21 @@ class LibraryViewModel {
     func getCellViewModel(at indexPath: IndexPath) -> CustomBookCellViewModel {
         CustomBookCellViewModel(bookModel: booksList[indexPath.row])
     }
+    
+    // API request
+    func getBooks() {
+        isLoading = true
+        
+        let onSuccess: ([Book]) -> Void = { books in
+            self.isLoading = false
+            self.booksList = books
+        }
+        
+        let onError: (Error) -> Void = { error in
+            self.isLoading = false
+            print(error)
+        }
+        
+        bookRepository.fetchBooks(onSuccess: onSuccess, onError: onError)
+    }
```

Comment:
You have the idea about how to reload the tableView using reloadTableView, but if you had more than one request and action to do, then you are going to have a lot of properties to handle them.
To avoid this you could use a closure: In `getBooks()` get a closure param and call it after `self.booksList = books`. Also you could get a closure to show an error and call it in `onError`